### ISSUE TITLE
Read Kristina's MTE ECTO ms

### ISSUE THREAD

**Issue by skmorgane** [weecology/mte-ecto#15]
**Status**: read. Talked with Kristina about jointly estimating MTE parameters from one data set & temperature's effects on metabolic rates from the other.

**Comment by KristinaRiemer** [comment]
Do you mean you talked to Dave?

**Comment by davharris** [comment]
@KristinaRiemer yeah, the "status" thing came from a different part of Github and made more sense in that context.  I'll have some more comments for you soon (tonight).

**Comment by davharris** [comment]
I was confused by the "Calculation of metabolic rates" section.  Should make clear:

* that there are two data sets. One (observational?) is about size, temperature, and metabolic rate; this one was used to determine parameters for MTE equation.  The other experimental one 

* Which parameters are being estimated The White et al. paper seemed to say that r_0 had to be estimated from the data for each species, if I read it correctly?

I think "non-STR" is a confusing name, because it says what it *isn't* rather than what it is.  "Constant size" seems like a clearer name, unless I'm missing something

Line 200: likelihood ratios were used to determine the significance of the effects, not their influence on the log-ratio, I think?

Line 202: I don't think it's clear what nonlinearity has to do with including starting temperature as a fixed effect? Also, I thought these coefficients were being estimated on the log scale?

Line 238: Doesn't the non-STR rate come from a model? If it comes from the linear mixed model, then shouldn't it be in the next section?

Line 244: degrees of freedom for t are usually shown as subscripts

I like the framing around line 280 about opposing forces making it unclear what the outcome will be unless both effects are included in predictions.  I feel like this angle could be played up more throughout the paper to help focus the reader on this question.

General thoughts:

* Is it correct to say that STR metabolic rates track the data, and non-STR rates are purely hypothetical? I feel like that could be emphasized more (possibly in the way you refer to the rates throughout, eg "the hypothetical metabolic rates without any STR").

* In the figures, it looks like color marks temperature and size marks mass.  Is there a way to mark the metabolic rate as well? 

* Figure 4: I thought "mass change for temperature to have no effect on metabolic rate" was hard to parse.  Maybe "mass change needed to balance thermal effect" or something?

Hope this is useful!

**Comment by KristinaRiemer** [comment]
The ms was revised several times and submitted May 17 with approval from all coauthors.
